benchmarks/perturbations.py

```diff
@@ -0,0 +1,66 @@
+import functools
+
+import numpy as np
+
+from astropy import units as u
+from astropy.time import Time
+from astropy.coordinates import Angle, solar_system_ephemeris
+
+from poliastro.twobody.propagation import cowell
+from poliastro.core.elements import rv2coe
+from poliastro.ephem import build_ephem_interpolant
+
+from poliastro.core.util import norm
+from poliastro.core.perturbations import (
+    J2_perturbation, J3_perturbation, atmospheric_drag, third_body, radiation_pressure
+)
+from poliastro.bodies import Earth, Moon, Sun
+from poliastro.twobody import Orbit
+
+
+def time_J2_propagation_Earth():
+    r0 = np.array([-2384.46, 5729.01, 3050.46])  # km
```

Comment:
With this structure, if the setup takes a significant amount of time, we will get noisy results. We should move everything that is not `cowell(...)` to a setup function:

https://asv.readthedocs.io/en/latest/writing_benchmarks.html#setup-and-teardown-functions